src/resources/projects/website/navigation/quarto-nav.scss

```diff
@@ -872,26 +872,46 @@ body .nav-footer {
   padding-right: 0.6em;
 }
 .nav-footer-left {
-  flex: 1 1 0px;
-  text-align: left;
+  @include media-breakpoint-up(md) {
+    flex: 1 1 0px;
+  }
+  @include media-breakpoint-down(sm) {
+    margin-bottom: 1em;
+    text-align: left;
+  }
+  @include media-breakpoint-down(sm) {
+    margin-bottom: 1em;
+  }
```

Comment:
@dragonstyle This seems to be duplicated.
Also, I believe `text-align: left;` is misplaced.
Left footer should always be left aligned by default.
This  currently causes the left footer to be center on large viewport.

Reply:
- Small viewport:
  <img width="730" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/85d60b57-5ca1-424a-b866-85dd9e19d757">

- Large viewport: the left part of the footer is centered.
  <img width="1079" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/e6f4172b-10ea-476b-92f1-2f6839de1130">

Reply:
I made a PR:
- #8458